ms/MSSel/MSAntennaParse.h

```diff
@@ -174,6 +174,7 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
   public:
     static MSAntennaParse* thisMSAParser;
     static MSSelectionErrorHandler* thisMSAErrorHandler;
+    static void cleanupErrorHandler() {if (thisMSAErrorHandler) delete thisMSAErrorHandler;thisMSAErrorHandler=0x0;}
```

Comment:
Please read my earlier question.   Without a coding guideline for CASACore
(not casa) I am not prepared to get into a protracted discussion here with
code reviewers reviewing code snippets without learning large design and/or
participating code debugging.  Coding guidelines are constraints on both,
the reviewer and code authors/maintainers.

If you don't want to merge this, leave a comment of why you won't merge and
leave it at that and take it up with others.  I am going to leave it at
this and the bug this fixes.

Thanks for your time so far.

On Wed, Sep 30, 2015 at 10:33 AM, Julian Taylor <notifications@github.com>
wrote:

> In ms/MSSel/MSAntennaParse.h
> <https://github.com/casacore/casacore/pull/252#discussion_r40817262>:
>
> > @@ -174,6 +174,7 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
> >    public:
> >      static MSAntennaParse* thisMSAParser;
> >      static MSSelectionErrorHandler* thisMSAErrorHandler;
> > +    static void cleanupErrorHandler() {if (thisMSAErrorHandler) delete thisMSAErrorHandler;thisMSAErrorHandler=0x0;}
>
> please elaborate on how this makes debugging easier. If it were splitted
> into multiple lines I could see it, but not as written.
>
> I'm not aware of coding style guidelines for casa, it would be good to get
> everything into a consistent shape but it would also be pretty annoying for
> people who are not used to others scrutinizing their code, it leads to lots
> of revisions before something gets merged.
> But coding guidelines should not be discussed here, that should go into a
> new issue.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/casacore/casacore/pull/252/files#r40817262>.[image:
> Web Bug from
> https://github.com/notifications/beacon/AM3Q7UZ6yjEST_Ga190yMRYj9elw4gX7ks5o3AZwgaJpZM4GGG24.gif]
>



-- 
संजय